.github/workflows/safetest.yml

```diff
@@ -7,41 +7,76 @@ on:
   push:
     branches:
       - main
+  workflow_dispatch:
+
 jobs:
   Safetest-Action:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
```

Comment:
Adding a way to run safetest jobs against multiple versions of node to ensure backwards compatibility.